client/blocks/edit-gravatar/test/index.jsx

```diff
@@ -42,13 +42,14 @@ describe( 'EditGravatar', function() {
 		};
 	} );
 
-	before( function() {
+	before( function( done ) {
```

Comment:
After some digging I think that loading the modules inside this `before()` are just taking a long time to `require()` for some reason, which intermittently causes the tests to fail. I might just remove the `done` hack and create a separate issue for this.